kinetic/distribution.yaml

```diff
@@ -5777,6 +5777,16 @@ repositories:
       url: https://github.com/ros-perception/perception_pcl.git
       version: kinetic-devel
     status: maintained
+  pheeno_ros:
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/ACSLaboratory/pheeno_ros-release.git
+      version: 0.1.1-1
+    source:
+      type: git
+      url: https://github.com/ACSLaboratory/pheeno_ros.git
+      version: 0.1.1
```

Comment:
The version tag of the source entry should be a valid branch name and not a tag. That would allow you to have automatic CI jobs each time a new commit is pushed to the branch

Reply:
Oh ok, I will fix that now!